caravel/viz.py

```diff
@@ -177,7 +177,7 @@ def get_df(self, query_obj=None):
                 if self.datasource.offset:
                     df.timestamp += timedelta(hours=self.datasource.offset)
         df.replace([np.inf, -np.inf], np.nan)
-        df = df.fillna(0)
+        df = df.fillna('NULL')
```

Comment:
df.fillna(None) threw an error 'must specify a fill method or value' 
But I get your point - if ('NULL') is treated as true in javascript. we may need some similar handling in javascript (like value=='NULL'?value:null).